uoguelph-engg4450/F24-kestra
# 4637-Tooltip-Out-Of-Bounds UI FIX --- Closes #4637

## What changes are being made and why?



The changes made allow for the fixing of the tooltip UI option. This changes it so that the words do not wrap when they are too long, rather they are inline no matter the language or the sizing of the screen. Please find below screenshots of this process.

---

### Before and After:
#### English
- Before
![Before](https://github.com/user-attachments/assets/01c364c1-0228-42fd-9a30-818b7ee795aa)
- After
![After](https://github.com/user-attachments/assets/bf15f823-2758-4a9f-b1da-e9385dc47062)

#### Russian
- Before
![Before](https://github.com/user-attachments/assets/0a5cc4ee-4979-4d26-873f-53432ee1ce04)
- After
![After](https://github.com/user-attachments/assets/f0d7f2e8-ea19-40de-93ab-1a267a586889)

#### Polish
- Before
![Before](https://github.com/user-attachments/assets/c6f84538-a570-4656-99ca-0e9bcc7a9312)
- After
![After](https://github.com/user-attachments/assets/3580d45d-7577-4116-a375-55b0556d853a)


---

Test Cases:
![English Test](https://github.com/user-attachments/assets/93a85409-e82c-4d98-84bc-5db47464ffbc)
![Russian](https://github.com/user-attachments/assets/1eca81d2-adb6-49b7-b2fa-927eb4f9c90b)
![Polish](https://github.com/user-attachments/assets/8f31b4cb-fb76-4b0d-bb59-cbb99d704086)

---

### Testing Instructions

1. Run `docker-compose up`
2. Run `npm run dev`
3. Run `npx cypress run`
4. Select _E2E Testing_
5. Click _Start E2E Testing_ in preferred browser
6. Click dashboard.cy.js
7. Observe test cases
